src/read/elf/symbol.rs

```diff
@@ -309,6 +309,14 @@ where
     pub(super) symbol: &'data Elf::Sym,
 }
 
+impl<'data, 'file, Elf: FileHeader, R: ReadRef<'data>> ElfSymbol<'data, 'file, Elf, R> {
+    /// Return a reference to the raw symbol structure.
+    #[inline]
+    pub fn as_sym(&self) -> &'data Elf::Sym {
```

Comment:
```suggestion
    pub fn raw_symbol(&self) -> &'data Elf::Sym {
```
For consistency with [COFF](https://github.com/gimli-rs/object/blob/57540508ab8654cec7ca20fa6609929785d1286b/src/read/coff/symbol.rs#L290). I may deprecate and rename later if I decide on a better naming scheme.